com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/flowable/FlowableFacade.java

```diff
@@ -328,4 +330,20 @@ public boolean isProcessInstanceSuspended(String processInstanceId) {
         return processInstance != null && processInstance.isSuspended();
     }
 
+    public void setSecondsToWaitOnShutdown(long secondsToWaitOnShutdown) {
+        LOGGER.info(format(Messages.FLOWABLE_SET_SECONDS_TO_WAIT_ON_SHUTDOWN, secondsToWaitOnShutdown));
+        AsyncExecutor asyncExecutor = processEngine.getProcessEngineConfiguration().getAsyncExecutor();
+        ((DefaultAsyncJobExecutor) asyncExecutor).setSecondsToWaitOnShutdown(secondsToWaitOnShutdown);
+    }
+
+    public long getSecondsToWaitOnShutdown() {
+        AsyncExecutor asyncExecutor = processEngine.getProcessEngineConfiguration().getAsyncExecutor();
+        return ((DefaultAsyncJobExecutor) asyncExecutor).getSecondsToWaitOnShutdown();
+    }
+
+    public void shutdown() {
```

Comment:
First, rename to shutdownProcessEngine
Second, I expected that the method will accept some configuration for shutdown. It will be more readable and understandable for a person who sees the code for the first time.